scripts/integrity.js

```diff
@@ -75,4 +77,4 @@ for (var i = 0; i < config.fontawesome.length; i++) {
     }
 }
 
-fs.writeFileSync(configFile, yaml.dump(config));
+fs.writeFileSync(configFile, yaml.dump(config, { lineWidth: 110 }));
```

Comment:
Please check the whole commit for this patch.

On Aug 5, 2016 00:39, "Joshua Mervine" <notifications@github.com> wrote:

> In scripts/integrity.js
> <https://github.com/MaxCDN/bootstrap-cdn/pull/727#discussion_r73607846>:
>
> > @@ -75,4 +77,4 @@ for (var i = 0; i < config.fontawesome.length; i++) {
> >      }
> >  }
> >
> > -fs.writeFileSync(configFile, yaml.dump(config));
> > +fs.writeFileSync(configFile, yaml.dump(config, { lineWidth: 110 }));
>
> What's this do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MaxCDN/bootstrap-cdn/pull/727/files/bc06fe1d74c5cbf81dc4ec2cda7c998c2c684dd3#r73607846>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAVVtWmVHGqiziUTl2sLQaZVEvv_M2r_ks5qclwPgaJpZM4JcmoI>
> .
>